sunpy/net/jsoc/jsoc.py

```diff
@@ -291,7 +291,7 @@ def search(self, *query, **kwargs):
 
         *Example 2*
 
-        Request keyword data of ``hmi.v_45s`` for certain specific keywords only::
+        Request keyword datJSOCResponsea of ``hmi.v_45s`` for certain specific keywords only::
```

Comment:
Is this a mistake?

Reply:
I'm sorry. I'll ensure this won't happen in future.

Reply:
It's fine.